admin/server/app/createStaticRouter.js

```diff
@@ -11,33 +11,50 @@ var express = require('express');
 var less = require('less-middleware');
 var path = require('path');
 var str = require('string-to-stream');
+var compose = require('../../../fields/utils/compose');
 
-function buildFieldTypesStream (fieldTypes) {
-	var src = '';
-	var types = Object.keys(fieldTypes);
-	['Column', 'Field', 'Filter'].forEach(function (i) {
-		src += 'exports.' + i + 's = {\n';
-		types.forEach(function (type) {
-			if (typeof fieldTypes[type] !== 'string') return;
-			src += type + ': require("../../fields/types/' + type + '/' + fieldTypes[type] + i + '"),\n';
+function buildFieldTypesStream (next) {
+	return (fieldTypes) => {
+		var types = Object.keys(fieldTypes);
+		var sections = { Columns: {}, Fields: {}, Filters: {} };
+		['Column', 'Field', 'Filter'].forEach(i => {
+			types.forEach(type => {
+				sections[i + 's'][type] = '../../fields/types/' + type + '/' + fieldTypes[type] + i;
+			});
+			if (i === 'Column') {
+				sections[i + 's'].id = '../../fields/components/columns/IdColumn';
+				sections[i + 's'].__unrecognised__ = '../../fields/components/columns/InvalidColumn';
+			}
```

Comment:
is there a good reason why you chose e.g `Column` instead of `Columns` in your iteration?
You could avoid concatenate the key-value with an `s` if you named your keys accordingly

```js
var sections = ['Columns', 'Fields', 'Filters'].reduce((acc, key) => {
  acc[key] = {}
  // ... your logic ...
  return acc
}, {})

return next(sections)
```